applications/openshift/high-availability/infra_nodes_in_two_zones_or_more/rule.yml

```diff
@@ -0,0 +1,54 @@
+documentation_complete: true
+
+title: 'Ensure infrastructure nodes are distribute across three failure zones'
+
+description: |- 
+    Distributing Kubernetes infrastructure nodes across failure zones enhances security by mitigating
+    the risk of a single point of failure and reducing the impact of infrastructure service
+    inavailability or attacks targeting a specific zone.
+
+rationale: |-
+    Distributing Kubernetes infrastructure nodes across failure zones is crucial for enhancing overall 
+    system resilience and security. By spreading infrastructure components across different zones, 
+    the system becomes more fault-tolerant, reducing the risk of widespread outages due to failures or 
+    attacks in a single zone. 
+    For infrastructure nodes, two failure zones are often deemed sufficient due to the nature of 
+    infrastructure services, which typically don't rely on quorum-based applications like etcd. 
+    Infrastructure services such as Ingress load balancers, logging or monitoring systems usually
+    operate independently across zones without requiring coordination for data consistency or failover. 
+    Failure zones are marked on nodes using a well-known label called "topology.kubernetes.io/zone". 
+    This label is automatically assigned to each node by cloud providers but might need to be managed
+    manually in other environments
+
+identifiers: {}
```

Comment:
unique identifier is missing, add it by using 
`./utils/rule_dir_json.py` followed by `python utils/fix_rules.py --product products/ocp4/product.yml add-cce --cce-pool redhat infra_nodes_in_two_zones_or_more`